app/services/DynamoChannelTests.scala

```diff
@@ -256,12 +256,16 @@ class DynamoChannelTests(stage: String, client: DynamoDbClient) extends DynamoSe
         .key(buildKey(channel, testName))
         .updateExpression("set lockStatus = :lockStatus")
         .expressionAttributeValues(Map(
-          ":lockStatus" -> jsonToDynamo(lockStatus.asJson)
+          ":lockStatus" -> jsonToDynamo(lockStatus.asJson),
         ).asJava)
+        .expressionAttributeNames(Map("#name" -> "name").asJava)
```

Comment:
we have to alias `name` to `#name` here because `name` is a reserved word in dynamodb!